api/path_management_status.go

```diff
@@ -93,14 +100,18 @@ func UpdateManagementStatus(nmStatus managementStatusInput, errorHandler ErrorHa
 		
 		// Update the NMP status in the local db
 		if err := persistence.SaveOrUpdateNMPStatus(db, fullName, *managementStatus); err != nil {
+			eventlog.LogNodeEvent(db, persistence.SEVERITY_ERROR, persistence.NewMessageMeta(EL_API_ERR_SAVE_NMP_STATUS, pDevice.Org, nmpName, err), persistence.EC_ERROR_NMP_STATUS_UPDATE, pDevice.Id, pDevice.Org, pDevice.Pattern, pDevice.Config.State)
 			return errorHandler(NewSystemError(fmt.Sprintf("Unable to update node management status object, error %v", err))), "", nil
 		} 
 
 		// Update the status of the NMP in the exchange
 		// if _, err := statusHandler(pDevice.Org, pDevice.Id, nmpName, managementStatus); err != nil {
-		// 	return errorhandler(NewSystemError(fmt.Sprintf("Unable to update node management status object in the exchange, error %v", err))), "", nil
+		// 	eventlog.LogNodeEvent(db, persistence.SEVERITY_ERROR, persistence.NewMessageMeta(EL_API_ERR_SAVE_NMP_STATUS_EXCHANGE, pDevice.Org, nmpName, err), persistence.EC_ERROR_NMP_STATUS_UPDATE, pDevice.Id, pDevice.Org, pDevice.Pattern, pDevice.Config.State)
+		// 	return errorHandler(NewSystemError(fmt.Sprintf("Unable to update node management status object in the exchange, error %v", err))), "", nil
 		// }
 
+		eventlog.LogNodeEvent(db, persistence.SEVERITY_INFO, persistence.NewMessageMeta(EL_API_COMPLETE_NMP_STATUS_UPDATE, pDevice.Org, nmpName), persistence.EC_NMP_STATUS_UPDATE_COMPLETE, pDevice.Id, pDevice.Org, pDevice.Pattern, pDevice.Config.State)
```

Comment:
Please change this one to something like "node management status for <nmp> changed to <new status>"